modules/Microsoft.ContainerService/managedClusters/readme.md

```diff
@@ -384,100 +385,100 @@ module managedClusters './Microsoft.ContainerService/managedClusters/deploy.bice
   name: '${uniqueString(deployment().name, location)}-test-csmaz'
   params: {
     // Required parameters
-    name: '<<namePrefix>>csmaz001'
```

Comment:
It apperas you're not using the latest PowerShell version. With the latest version, a breaking change to the `ConvertFrom-Json` command was introduced that highly impacts how the `Set-ModuleReadMe` script sorts the properties of the examples in the readme. They should be alphabetical. Couly you check and re-run again?

Reply:
Thanks for the heads up. Will fix this.